loleaflet/src/control/Control.MobileTopBar.js

```diff
@@ -125,8 +125,26 @@ L.Control.MobileTopBar = L.Control.extend({
 			w2ui['actionbar'].hide('acceptformula', 'cancelformula');
 			w2ui['actionbar'].show('undo', 'redo');
 		}
-		else if (id === 'insertcomment') {
-			this.map.insertComment();
+		else if (id === 'comment_wizard') {
+			if (window.commentWizard) {
+				window.commentWizard = false;
+				var map = this._map;
+				$('.ui-header.level-0.mobile-wizard').each(function() {
+					map._docLayer._removeHighlightSelectedWizardComment(this.annotation);
+				});
+				this.map.fire('closemobilewizard');
+				toolbar.uncheck(id);
+			}
+			else {
+				if (window.insertionMobileWizard)
+					this.onClick(null, 'insertion_mobile_wizard');
+				else if (window.mobileWizard)
+					this.onClick(null, 'mobile_wizard');
+				window.commentWizard = true;
+				var menuData = L.annotationManager(this.map).getCommentWizardStructure();
```

Comment:
this line creates new instance of AnnotationManager, which during initialization adds 'AnnotationReply' listener so later it receives notifications on every map.fire('AnnotationReply') and couses multiple tiggers.